soc/arm/arm/fvp_aemv8r_aarch32/CMakeLists.txt

```diff
@@ -3,3 +3,5 @@
 
 zephyr_library_sources_ifdef(CONFIG_ARM_MPU arm_mpu_regions.c)
 zephyr_library_sources(soc.c)
+
+set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/include/zephyr/arch/arm/cortex_a_r/scripts/linker.ld CACHE INTERNAL "")
```

Comment:
I wonder if this support "includes" multiple ".ld" files in case we need it? If yes, how we should do, by just appending a new linker file to `SOC_LINKER_SCRIPT`?

Reply:
as of now, it doesn't.

But when going through the linker scripts I did notice quite a few which could benefit from such support.
But I would see that as further improvement / cleanup on top of this PR, together with a couple of other possible cleanups.

Reply:
Sure, totally agree with your plan. It's just a simple question about the feature status